### ISSUE TITLE
Freeze ssb-dbgen version

### ISSUE THREAD

**Issue by GindaChen** [UWHustle/hustle#113]
Got the following error due to an update to the ssb-dbgen:
```
> sh ./scripts/ssb/gen_benchmark_data.sh 1
...
CMake Error at CMakeLists.txt:163 (set_property):
  C_STANDARD property may not be appended.


CMake Error at CMakeLists.txt:163 (set_property):
  C_STANDARD property may not be appended.
```

@srsuryadev I think we can either
1. Upgrade CMake to 3.20, or
2. Freeze a ssb-gen version

**Comment by srsuryadev** [comment]
I think we can upgrade CMake to 3.20

**Comment by GindaChen** [comment]
Fixed by #115
